README.md

```diff
@@ -2,16 +2,13 @@
 
 Shows disassembly to source relations and dims unused source lines.
 
-This extension is based on [Compiler Explorer](https://github.com/mattgodbolt/compiler-explorer)
-(especially on AsmParser class from it).
+This extension is based on [Compiler Explorer](https://github.com/mattgodbolt/compiler-explorer) (especially on AsmParser class from it).
```

Comment:
Also it is always better to leave lines unrelated to change in their original form.
```suggestion
This extension is based on [Compiler Explorer](https://github.com/mattgodbolt/compiler-explorer)
(especially on AsmParser class from it).
```